src/jobs/native_runtime.html

```diff
@@ -0,0 +1,104 @@
+---
+title: Dart Native Runtime Engineer
+---
+
+<p>The Flutter/Dart runtime team is looking to improve the performance,
+   size and stability provided by the Flutter/Dart Native runtime.
+   To that end, we’re seeking software engineers to join our team and
+   contribute to this effort.</p>
+
+<p>In the “Dart Native Runtime Engineer” role, you’ll be part
+  of a team that is tasked with the following mandates:</p>
+
+<ul>
+  <li>Provide a Dart Virtual Machine runtime that has 
+      the ability to execute Dart code efficiently.
+  </li>
+  <li>Debug and profile Flutter/Dart applications on complex setups.
+  </li>
+  <li>Collaborate closely with Flutter Engine and Framework teams to improve
+      both development and deployment of Flutter applications. 
+  </li>
+  <li>Triage and address issues to make Dart/Flutter apps stable and delightful.
+  </li>
+</ul>
+
+<h2>Job responsibilities</h2>
+
+<ol type="A">
+  <li>Own or assist with one or more of Dart's 
+      virtual machine subsystems.
+  </li>
+  <li>Work with the Flutter team to ensure Flutter apps are performant 
```

Comment:
Maybe: "Work with customers and the Flutter team to ensure apps are performant"?

Reply:
Done.